topics/single-cell/tutorials/scrna_batch_correction/tutorial.md

```diff
@@ -0,0 +1,529 @@
+>
+> The rest of the workflow will be the same as shown in this tutorial, but you will end up with different results because `SCTransform` handles preprocessing in a slightly different way than the three separate tools. If you want to learn more about these differences then you can choose the SCTransform route in the [Clustering 3k PBMCs with Seurat]({% link topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.html %}) tutorial.
+{: .comment}
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Integrate** {% icon tool %})
+>    - *"Method used"*: `Normalize with 'NormalizeData'`
+>        - *"Method for normalization"*: `LogNormalize`
+>
+> 2. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Identify highly variable genes with 'FindVariableFeatures'`
+>        - *"Method to select variable features"*: `vst`
+>        - *"Output list of most variable features"*: `No`
+>
+> 3. {% tool [Seurat Preprocessing](toolshed.g2.bx.psu.edu/repos/iuc/seurat_preprocessing/seurat_preprocessing/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Scale and regress with 'ScaleData'`
+>        - *"Regress out a variable"*: `No`
+>        - *"Features to scale"*: `Variable Features`
+>
+> 4. {% tool [Seurat Run Dimensional Reduction](toolshed.g2.bx.psu.edu/repos/iuc/seurat_reduce_dimension/seurat_reduce_dimension/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Preprocessing** {% icon tool %})
+>    - *"Method used"*: `Run a PCA dimensionality reduction using 'RunPCA'`
+>
+>    > <comment-title> short description </comment-title>
+>    >
+>    > We will use the output from `RunPCA` in the  following section when we perform batch correction.
+>    >
+>    > If you're already very familiar with the Seurat clustering pipeline and you just want to try using the  {% tool Seurat Integrate %} tools, then you can skip ahead to the **Clustering after Integration** step now.
```

Comment:
```suggestion
>    > If you're already familiar with the Seurat clustering pipeline and you just want to try using the  {% icon tool %} **Seurat Integrate** tools, then you can skip ahead to the **Clustering after Integration** step now.
```

Reply:
Do these 'skips' work for running through the tutorial? Or is that assuming they won't actually be using Galaxy while reading the tutorial?